### PR TITLE
{lang}[intel/2015a] R 3.1.3

### PR DESCRIPTION
**Pull request by boegel**
synced with #1496

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/0c5c4f036700b80657fd for a full test report.

**Comment by boegel** [comment]
@wpoely86: please review?

File: easybuild/easyconfigs/r/R/R-3.1.3-intel-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,381 @@
...
+    ('reshape', '0.8.5', ext_options),
+    ('RNASeqPower', '1.6.0', bioconductor_options),
+    ('VennDiagram', '1.6.9', ext_options),
+    ('xlsxjars', '0.6.1', ext_options),
+    ('xlsx', '0.5.7', ext_options),
+    ('Rsubread', '1.16.1', bioconductor_options),
+]
+
+moduleclass = 'lang'
+
```
Comment (wpoely86) [inline easybuild/easyconfigs/r/R/R-3.1.3-intel-2015a.eb]: drop newline?

**Comment by pescobar** [comment]
Test report by @pescobar
**FAILED**
Build succeeded for 6 out of 8 (1 easyconfigs in this PR)
Linux centos 6.5, Intel(R) Xeon(R) CPU E5-2620 0 @ 2.00GHz, Python 2.6.6
See https://gist.github.com/df18747528d9c50a7916 for a full test report.

**Comment by boegel** [comment]
@pescobar: Java installation failed most likely due to missing 'sources', please feed in the Java 'source' tarball and retry?

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
Linux centos 6.5, Intel(R) Xeon(R) CPU E5-2620 0 @ 2.00GHz, Python 2.6.6
See https://gist.github.com/699d84e122e057180f2c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/d36911caa40a10debe53 for a full test report.

**Comment by pescobar** [comment]
@boegel it looks that when the easyconfig contains so many extensions the generated log file is not complete. 

As example, I tried to do "grep External easybuild/easybuild-R-3.1.3-20150323.183313.log" to verify which version of BLAS is used but I get no output. I could not find the line mentioned here https://github.com/hpcugent/easybuild-easyconfigs/issues/1435

Maybe I should report this as an independent issue?

**Comment by boegel** [comment]
@pescobar: you may be running into this: https://github.com/hpcugent/easybuild-framework/issues/1094

**Comment by boegel** [comment]
Synced up with #1496, going in, fix for the incomplete log is being looked into via https://github.com/hpcugent/easybuild-framework/issues/1238

Thanks for testing @pescobar and reviewing @wpoely86!

**Pull request merged**